ddpui/api/user_org_api.py

```
@@ -378,7 +378,23 @@
+    _, error = orguserfunctions.request_reset_password(payload.email, False)
+    if error:
+        raise HttpError(400, error)
+    return {"success": 1}
+
+
+@user_org_router.post(
+    "/users/forgot_password_v2/",
+    auth=None,
+)
+def post_forgot_password_v2(
+    request, payload: ForgotPasswordSchema
```

Ruff (0.12.2)

Unused function argument: `request`

(ARG001)